src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,46 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
```

Comment:
```suggestion
    <li className="PostInfo">
```
use semantic tags for a list

Reply:
the same with `Comments`